test/integration/options.spec.js

```diff
@@ -346,6 +346,26 @@ describe('options', function() {
           done();
         });
       });
+
+      it('fails if no --grep', function(done) {
+        args = ['--invert'];
+        runMocha(
+          'options/grep.fixture.js',
+          args,
+          function(err, res) {
+            if (err) {
+              done(err);
+              return;
+            }
+            expect(res, 'to satisfy', {
+              code: 1,
+              output: /fgrep/
```

Comment:
@craigtaub, where's the corresponding test for invalid reporter option from previous PR?

Reply:
Not sure one was added, might have been missed. If not we should add one.